backend/src/libs/utils/replaceChar.ts

```diff
@@ -0,0 +1,3 @@
+export const replaceChar = (text: string) => {
+  return text.replace(/[a-z0-9'#]/g, 'a').replace(/[A-Z]/g, 'A');
```

Comment:
I believe these 2 replaces are not necessary. Only one. As no one cares if there was a capital letter if it's "hidden" as a bunch of a's.

```suggestion
  return text.replace(/[A-Za-z0-9'#]/g, 'a')
```

Reply:
@comoser I think when you replace the characters on user name, maybe you can maintain the upper case letters.
I don't know, is only a suggestion.